catalogs/catalog-lakehouse-iceberg/src/main/java/org/apache/gravitino/catalog/lakehouse/iceberg/IcebergCatalogOperations.java

```diff
@@ -513,6 +514,13 @@ public Table createTable(
                           .build())
               .toArray(IcebergColumn[]::new);
 
+      // Gravitino NONE distribution means the client side doesn't specify distribution not the same
```

Comment:
distribution not the -> distribution, which is not the same as none distribution in Iceberg.

Reply:
updated